fs.go

```diff
@@ -73,7 +73,7 @@ func writeFile(path string, in toml.Marshaler) error {
 		return err
 	}
 
-	_, err = f.Write(s)
+	_, err = f.Write(append([]byte(comment), s...))
```

Comment:
It's pretty minor, but creating a new byte slice twice here (once to convert the string, another time to append) seems like overkill. May be better to define `comment` as a `[]byte` var and instead of `append` just call `f.Write()` twice